### PR TITLE
Update running-testnet-node.md

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
## Description

Describe the changes that were made in this pull request. When possible start with the motivations behind the change. Be sure to also include the following information:

1. Motivation for change?
2. What was changed?
3. Link to relevant issues?

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

### CODE REVIEW

**Comment by LakshmiLavanyaKasturi** [comment]
@pgray-hiro: Could you review my PR? Thanks.

**Comment by LakshmiLavanyaKasturi** [comment]
@sabbyanandan: Could you review my edits. Thanks.

File: docs/get-started/running-testnet-node.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ In order for the API node services to work correctly, you must configure any net
 the ports discussed in this section. The details of network and firewall configuration are highly specific to your
 machine and network, so a detailed example isn't provided.
 
-The following ports must open on the host machine:
+You can open the following ports on the host machine:
```
Comment (sabbyanandan) [inline docs/get-started/running-testnet-node.md:31]: The firewall configuration described in this section is a requirement. In other words, the ports must be open and accessible for the node to receive and sync data.

Considering that, it'd be better to state the explicit requirement as-is. Perhaps: "The following ports must be open on the host machine:"

**Review by sabbyanandan — CHANGES_REQUESTED** [review]

File: docs/get-started/running-testnet-node.md (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docs/get-started/running-testnet-node.md:31]: Sounds good!

**Review by sabbyanandan — APPROVED** [review]
LGTM

**Pull request merged**